NanoXLSX/LowLevel/XlsxReader.cs

```diff
@@ -70,32 +70,81 @@ public XlsxReader(Stream stream, ImportOptions options = null)
         /// </exception>
         public void Read()
         {
-            try

```

Comment:
The try block as wrapper is still necessary since FileStream could fail with several other Exceptions than a common IOException (An own version is used anyway --> Exceptions.IOException). The only Exception that is used as interface is this mentioned IOException.
See also: [.NET documentation about possible exception for FileStream](https://learn.microsoft.com/en-us/dotnet/api/system.io.filestream.-ctor?view=netstandard-2.0#system-io-filestream-ctor(system-string-system-io-filemode))